job_spec/INSAR_ISCE_BURST.yml

```diff
@@ -0,0 +1,53 @@
+INSAR_ISCE_BURST:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 2
+        example:
+          - S1_249434_IW1_20230523T170733_VV_8850-BURST
+          - S1_249434_IW1_20230511T170732_VV_07DE-BURST
+        items:
+          description: The reference and secondary bursts.
```

Comment:
I think this is technically the description for an individual item, so "The name of the Sentinel-1 SLC IW burst granule to process" would be slightly more accurate